kura/distrib/src/main/resources/Win32-nn/start_kura_debug.bat

```diff
@@ -0,0 +1,32 @@
+@echo off
+
+set KURA_ROOT=c:\opt\eclipse
+set KURA_HOME=%KURA_ROOT%\kura\kura
+set KURA_HOME_URI=/%KURA_HOME:\=/%
+set KURA_DATA=%KURA_ROOT%\kura\data
+set KURA_TEMP=\tmp\.kura
```

Comment:
If UAC is active (high) and the batch file is not run as Admin, it will fail.
We expect those going to debug to have advanced knowledge.